### PR TITLE
[FE-20] Implement locale placeholder

### PR DESCRIPTION
**Pull request by pluma4345**
### Scope & Purpose

This PR changes the `LocaleInput` to take an additional `placeholder` property and changes the placeholders for each analyzer type to a correct format.

- [ ] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [x] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [x] Backport for 3.11: #19128
  - [x] Backport for 3.10: #19129
  - [ ] Backport for 3.9: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [x] GitHub issue / Jira ticket: FE-20
- [ ] Design document:

### CODE REVIEW

**Review by palashkaria — APPROVED** [review]
LGTM

**Comment by Dronplane** [comment]
One question - why for different analyzers locale placeholders looks different?  I'd expecte to have just one placeholder for any analyzers that use locale.

File: js/apps/system/_admin/aardvark/APP/react/src/views/analyzers/forms/NormForm.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const NormForm = ({ formState, dispatch, disabled }: FormProps<NormState>) =>
   <Grid>
     <Cell size={'1-2'}>
       <LocaleInput formState={formState} dispatch={dispatch as Dispatch<DispatchArgs<LocaleProperty>>}
-                   disabled={disabled}/>
+                   placeholder="language[_COUNTRY]" disabled={disabled}/>
```
Comment (Dronplane) [inline js/apps/system/_admin/aardvark/APP/react/src/views/analyzers/forms/NormForm.tsx:13]:
```suggestion
                   placeholder="language[_COUNTRY][_VARIANT]" disabled={disabled}/>
```

File: js/apps/system/_admin/aardvark/APP/react/src/views/analyzers/forms/TextForm.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ const TextForm = ({ formState, dispatch, disabled }: FormProps<TextState>) => {
       <Grid>
         <Cell size={'1'}>
           <LocaleInput formState={formState} dispatch={dispatch as Dispatch<DispatchArgs<LocaleProperty>>}
-                       disabled={disabled}/>
+                       placeholder="language[_COUNTRY]" disabled={disabled}/>
```
Comment (Dronplane) [inline js/apps/system/_admin/aardvark/APP/react/src/views/analyzers/forms/TextForm.tsx:40]:
```suggestion
                       placeholder="language[_COUNTRY][_VARIANT]" disabled={disabled}/>
```

**Review by Dronplane — CHANGES_REQUESTED** [review]
Checked the code. Actual usage should be like this. @Simran-B it would be also good to have same changes in the documentation

**Comment by pluma** [comment]
@Dronplane @Simran-B The PR is implemented in accordance with the placeholder formats specified in the ticket FE-20. Is the ticket wrong?

**Comment by Simran-B** [comment]
norm and text do indeed accept `_VARIANT`

**Review by ajurdak — APPROVED** [review]
LGTM.

**Review by Dronplane — APPROVED** [review]
LGTM

**Pull request merged**